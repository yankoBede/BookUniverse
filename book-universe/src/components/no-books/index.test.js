import React from 'react'
import renderer from 'react-test-renderer'
import NoBooks from './index'

describe('NoBooks Component', () => {
    it('should render No Books image and text', () => {
        const tree = renderer.create(
            <NoBooks/>
        ).toJSON()
        expect(tree).toMatchSnapshot()
    })
})