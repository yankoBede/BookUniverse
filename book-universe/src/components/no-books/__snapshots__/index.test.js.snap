// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoBooks Component should render No Books image and text 1`] = `
<div
  className="div"
>
  <img
    alt="mp-books"
    src="https://media.istockphoto.com/vectors/shelf-vector-id165794808?k=6&m=165794808&s=612x612&w=0&h=14iBSQo-gCeY-I6DicAu3dndy_vua4NXXnj6fCgNdTw="
  />
  <div>
    <h3
      className="h3"
    >
      No available books yet
    </h3>
  </div>
</div>
`;
